slides/02-data-budget.qmd

```diff
@@ -300,7 +300,7 @@ Use `strata = latency`
 
 ```{r}
 set.seed(123)
-frog_split <- initial_split(tree_frogs, strata = latency)
+frog_split <- initial_split(tree_frogs, prop = 0.8, strata = latency)
```

Comment:
I've added `prob = 0.8` back in because that's what we're using in the following deck